builder_data.yaml

```diff
@@ -50,10 +50,11 @@ apt_repo_command: |
   keychain -k mine;
 yum_repo_command: |
   for repodir in $(find "__REPO_PATH__" -mindepth 2 -name "*.rpm" | xargs -I {} dirname {} | uniq) ; do
-    [ -d "${repodir}" ] || continue;
-    echo "Generating repodata for ${repodir}..."
-    sudo createrepo --checksum=sha --database --update "${repodir}";
-    echo "Finished generating repodata for ${repodir}..."
+    [ -d "${repodir}" ] || continue &&
+    echo "Generating and signing repodata for ${repodir} with __GPG_KEY__..." &&
+    sudo createrepo --checksum=sha --database --update "${repodir}" &&
+    gpg --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml" &&
+    gpg --export --armour __GPG_KEY__ > "${repodir}/repodata/repomd.xml.key"
```

Comment:
If there's no centralized signing service, yeah. I think I found the script in question; in theory it could be hooked into a signing pipeline to ensure that keys get stripped before signing (if you were feeling constructive). I'm validating that it still does The Thing since I haven't touched it in a while.